src/App.js

```diff
@@ -20,25 +24,38 @@ export default function App() {
       .catch(error => console.error(error))
   }, [currentCity])
 
+  console.log(bookmarks)
+  console.log(concerts)
+
   return (
     <>
-      {activePage === 'suche' && (
-        <SearchPage
-          pageName="Suche"
-          concerts={concerts}
-          onNavigate={handleClickDetails}
-          onSubmit={chosenCity}
-        />
-      )}
+      <Switch>
+        <Route exact path="/suche">
```

Comment:
you don't need exact path here (or on the other pages)

```suggestion
        <Route  path="/suche">
```